astropy/wcs/wcsapi/utils.py

```diff
@@ -49,7 +49,7 @@ def wcs_info_str(wcs):
 
     # Find largest between header size and value length
     pixel_dim_width = max(9, len(str(wcs.pixel_n_dim)))
-    pixel_nam_width = max(9, max(len(x) for x in wcs.pixel_axis_names))
+    pixel_nam_width = max(9, *(len(x) for x in wcs.pixel_axis_names))
```

Comment:
An even shorter option:
```suggestion
    pixel_nam_width = max(9, *map(len, wcs.pixel_axis_names))
```